### ISSUE TITLE
Another MergerFS Failure

### ISSUE THREAD

**Issue by bifa2** [TechPerplexed/Gooby#195]
Hi,

Sorry for opening another issue, but the other cases haven't helped. I have tried:

1.
```
sudo systemctl daemon-reload
sudo systemctl stop mergerfs
sudo systemctl stop rclonefs
/bin/fusermount -uz /mnt/rclone
/bin/fusermount -uz /mnt/google

and after that try run rclean
```

2. Uninstalling, and reinstalling rclone. The output is this:
```
Removed /etc/systemd/system/default.target.wants/rclonefs.service.
Removed /etc/systemd/system/default.target.wants/mergerfs.service.
Removed /etc/systemd/system/unionfs.service.wants/rclonefs.service.
Removed /etc/systemd/system/mergerfs.service.wants/rclonefs.service.

--------------------------------------------------
 remove Rclone completed
--------------------------------------------------

---------------- install ------------------------

Please follow the instructions to setup Rclone

Current remotes:

Name                 Type
====                 ====
GDrive-Movies        drive
Local                local

e) Edit existing remote
n) New remote
d) Delete remote
r) Rename remote
c) Copy remote
s) Set configuration password
q) Quit config
e/n/d/r/c/s/q> q

Confirm that this is what you named your mount  GDrive-Movies

What is your media folder in GDrive-Movies?
Leave empty for root - not recommended! (ex: Media)

cat: /var/local/Gooby/Docker/.config/cf_email: No such file or directory
cat: /var/local/Gooby/Docker/.config/cf_key: No such file or directory
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100    14  100    14    0     0    111      0 --:--:-- --:--:-- --:--:--   111
done
Created symlink /etc/systemd/system/default.target.wants/rclonefs.service → /etc/systemd/system/rclonefs.service.
Created symlink /etc/systemd/system/unionfs.service.wants/rclonefs.service → /etc/systemd/system/rclonefs.service.
Created symlink /etc/systemd/system/mergerfs.service.wants/rclonefs.service → /etc/systemd/system/rclonefs.service.
Created symlink /etc/systemd/system/default.target.wants/mergerfs.service → /etc/systemd/system/mergerfs.service.
0,15,30,45 * * * * /opt/Gooby/scripts/cron/syncmount.sh > /dev/null 2>&1

 Done!


--------------------------------------------------
 install Rclone completed
--------------------------------------------------
```

but running rclean/gooby clean up just results in "waiting on mounts to be created..." again.

I'm stuck :(

**Comment by bifa2** [comment]
After a reboot, the containers aren't started. Trying to start via rclean results in the same message. 

I managed to get things up by running `docker-compose up -d` in `/var/local/Gooby/Docker` but ... everything will break again when anything tries to update.

**Comment by TechPerplexed** [comment]
Well let's see if we can find out what's the trouble. First, after you run

```
sudo systemctl daemon-reload
sudo systemctl stop mergerfs
sudo systemctl stop rclonefs
/bin/fusermount -uz /mnt/rclone
/bin/fusermount -uz /mnt/google
```

What folders do you see in `ls /mnt` ?

**Comment by bifa2** [comment]
Thanks for the quick reply. I'll try this tomorrow and get back to you - will leave the server as is this evening so my gf and I can watch a few things remotely at the same time 😅

**Comment by TechPerplexed** [comment]
No rush from my side, but I am not sure you'll be able to watch your stuff without a working mount ^_^

**Comment by mahogl** [comment]
I had the exact same problem and I had to manually delete   a folder under /mnt to get it working again.

**Comment by bifa2** [comment]
I get:

```
sudo systemctl daemon-reload
sudo systemctl stop mergerfs
sudo systemctl stop rclonefs
/bin/fusermount -uz /mnt/rclone
/bin/fusermount: entry for /mnt/rclone not found in /etc/mtab
/bin/fusermount -uz /mnt/google
/bin/fusermount: entry for /mnt/google not found in /etc/mtab
```

at this point, `ls /mnt` lists 'local' and 'uploads'.

I've a virtual snapshot of the server, so I'm gonna roll back now (as /mnt has the right mounts in that... it's just when I start stopping/restarting services that things break).

**Comment by TechPerplexed** [comment]
So when we put back the services one by one, what errors are you getting?

First

`sudo systemctl start rclonefs`

Check if /mnt/rclone shows your files, then

`sudo systemctl start mergerfs`

What is shown in /mnt/google? Getting any errors somewhere?

**Comment by bifa2** [comment]
Sorry for the delayed reply - I had to leave the server running (as I have a few users who use it and was away for some days, then I forgot about the issue until Plex was unreachable this morning). Gave the server a reboot and logged back in to re-run `docker-compose up` and it's back and content streams ok.

I'm going to try patching the server now and giving it a reboot and seeing if the various containers start up.

**Comment by bifa2** [comment]
All good it seems :)
